service/history/api/startworkflow/api.go

```diff
@@ -249,7 +250,11 @@ func (s *Starter) lockCurrentWorkflowExecution(
 // prepareNewWorkflow creates a new workflow context, and closes its mutable state transaction as snapshot.
 // It returns the creationContext which can later be used to insert into the executions table.
 func (s *Starter) prepareNewWorkflow(workflowID string) (*creationParams, error) {
-	runID := uuid.NewString()
+	runUUID, err := uuid.NewV7()
```

Comment:
there is another uuid.NewString() call in this same file, do you need to update that as well?

Reply:
I did, thank you.